frontend/dockerfile/docs/reference.md

```diff
@@ -2,159 +2,13 @@
- => [internal] load metadata for docker.io/library/alpine:3.2              0.4s
- => CACHED [1/2] FROM docker.io/library/alpine:3.2@sha256:e9a2035f9d0d7ce  0.0s
- => CACHED [2/2] RUN apk add --no-cache socat                              0.0s
- => exporting to image                                                     0.0s
- => => exporting layers                                                    0.0s
- => => writing image sha256:1affb80ca37018ac12067fa2af38cc5bcc2a8f09963de  0.0s
- => => naming to docker.io/svendowideit/ambassador                         0.0s
-```
-
-By default, the build cache is based on results from previous builds on the machine
-on which you are building. The `--cache-from` option also allows you to use a
-build-cache that's distributed through an image registry refer to the
-[specifying external cache sources](https://docs.docker.com/engine/reference/commandline/build/#specifying-external-cache-sources)
-section in the `docker build` command reference.
-
-When you're done with your build, you're ready to look into [scanning your image with `docker scan`](https://docs.docker.com/engine/scan/),
-and [pushing your image to Docker Hub](https://docs.docker.com/docker-hub/repos/).
-
-## BuildKit
-
-Starting with version 18.09, Docker supports a new backend for executing your
-builds that is provided by the [moby/buildkit](https://github.com/moby/buildkit)
-project. The BuildKit backend provides many benefits compared to the old
-implementation. For example, BuildKit can:
-
-- Detect and skip executing unused build stages
-- Parallelize building independent build stages
-- Incrementally transfer only the changed files in your build context between builds
-- Detect and skip transferring unused files in your build context
-- Use external Dockerfile implementations with many new features
-- Avoid side-effects with rest of the API (intermediate images and containers)
-- Prioritize your build cache for automatic pruning
-
-To use the BuildKit backend, you need to set an environment variable
-`DOCKER_BUILDKIT=1` on the CLI before invoking `docker build`. [Docker Buildx](https://github.com/docker/buildx)
-always enables BuildKit.
-
-### External Dockerfile frontend
-
-BuildKit supports loading frontends dynamically from container images. To use
-an external Dockerfile frontend, the first line of your Dockerfile needs to be
-`# syntax=docker/dockerfile:1` pointing to the specific image you want to use.
-
-BuildKit also ships with Dockerfile frontend builtin but it is recommended to
-use an external image to make sure that all users use the same version on the
-builder and to pick up bugfixes automatically without waiting for a new version
-of BuildKit or Docker engine.
-
-See the [`syntax` directive section](#syntax) for more information.
+> When you are done reading this page, refer to the [Packaging your software](https://docs.docker.com/build/building/packaging/)
```

Comment:
> When you are done reading this page

No one's going to read this whole page :laughing: it's a little long! We could just remove that bit.